package-lock.json

```diff
@@ -2724,7 +2724,7 @@
       "requires": {
```

Comment:
`package-lock.json` shouldn't be modified in this PR

Reply:
Removed